modules/preparing-an-inital-cluster-deployment-for-mce-connected.adoc

```diff
@@ -0,0 +1,297 @@
+// Module included in the following assemblies:
+//
+// * installing_with_agent_based_installer/preparing-an-agent-based-installed-cluster-for-mce.adoc
+
+:_content-type: PROCEDURE
+[id="preparing-an-inital-cluster-deployment-for-mce-connected_{context}"]
+
+= Preparing an agent-based cluster deployment for multicluster engine for Kubernetes (MCE) Operator while connected
+
+Create the required manifests for the multicluster engine for Kubernetes (MCE) Operator, Local Storage Operator (LSO), and to deploy an agent-based {product-title} cluster as a hub cluster.
+
+.Procedure
+
+. Create a sub-folder named  `openshift` in the `<assets_directory>` folder. This sub-folder is used to store the extra manifests that will be applied during the installation to further customize the deployed cluster.
+The `<assets_directory>` folder contains all the assets including the `install-config.yaml` and `agent-config.yaml` files.
+
++
+[NOTE]
+====
+The installer does not validate extra manifests.
+====
+
+. For MCE, create the following manifests and save them in the `<assets_directory>/openshift` folder:
+
++
+.Example `mce_namespace.yaml`
+
```

Comment:
In this file too, should remove the extra blank lines (I won't comment them all)
```suggestion
```